rules/RESPONSE-955-WEB-SHELLS.conf

```diff
@@ -0,0 +1,603 @@
+    phase:4,\
+    deny,\
+    capture,\
+    t:none,\
+    msg:'rusuh web shell',\
+    logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-multi',\
+    tag:'platform-iis',\
+    tag:'platform-windows',\
+    tag:'attack-disclosure',\
+    tag:'paranoia-level/1',\
+    tag:'OWASP_CRS',\
+    tag:'capec/1000/118/116',\
+    ctl:auditLogParts=+E,\
+    ver:'OWASP_CRS/3.3.0',\
+    severity:'CRITICAL',\
+    setvar:'tx.outbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}',\
+    setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
+
+# Safe0ver web shell
+SecRule RESPONSE_BODY "@contains Safe0ver Shell //Safe Mod Bypass  Safe0ver Shell" \
+    "id:955300,\
+    phase:4,\
+    deny,\
+    capture,\
+    t:none,\
+    msg:'Safe0ver web shell',\
+    logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-multi',\
+    tag:'platform-iis',\
+    tag:'platform-windows',\
+    tag:'attack-disclosure',\
+    tag:'paranoia-level/1',\
+    tag:'OWASP_CRS',\
+    tag:'capec/1000/118/116',\
+    ctl:auditLogParts=+E,\
+    ver:'OWASP_CRS/3.3.0',\
+    severity:'CRITICAL',\
+    setvar:'tx.outbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}',\
+    setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
+
+# Symlink_Sa web shell
+SecRule RESPONSE_BODY "@rx <title>Symlink_Sa [0-9.]+</title>" \
+    "id:955310,\
+    phase:4,\
+    deny,\
+    capture,\
+    t:none,\
+    msg:'Symlink_Sa web shell',\
+    logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-multi',\
+    tag:'platform-iis',\
+    tag:'platform-windows',\
+    tag:'attack-disclosure',\
+    tag:'paranoia-level/1',\
+    tag:'OWASP_CRS',\
+    tag:'capec/1000/118/116',\
+    ctl:auditLogParts=+E,\
+    ver:'OWASP_CRS/3.3.0',\
+    severity:'CRITICAL',\
+    setvar:'tx.outbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}',\
+    setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
+
+# SyRiAn Sh3ll web shell
+SecRule RESPONSE_BODY "@contains <title>SyRiAn Sh3ll ~ " \
+    "id:955320,\
+    phase:4,\
+    deny,\
+    capture,\
+    t:none,\
+    msg:'SyRiAn Sh3ll web shell',\
+    logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-multi',\
+    tag:'platform-iis',\
+    tag:'platform-windows',\
+    tag:'attack-disclosure',\
+    tag:'paranoia-level/1',\
+    tag:'OWASP_CRS',\
+    tag:'capec/1000/118/116',\
+    ctl:auditLogParts=+E,\
+    ver:'OWASP_CRS/3.3.0',\
+    severity:'CRITICAL',\
+    setvar:'tx.outbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}',\
+    setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
+
+# webadmin.php file manager
+SecRule RESPONSE_BODY "@contains <h1 style=\"margin-bottom: 0\">webadmin.php</h1>" \
```

Comment:
a very impressive collection! looks good :) Just worried about this line that seems too "generic" and could lead to some FP, isn't it?

Reply:
Yes, that one was tricky. Maybe you will be able to find a better pattern
[webadmin.php.txt](https://github.com/coreruleset/coreruleset/files/5766009/webadmin.php.txt)

Reply:
what if this line:
```
<img src="shell-webadmin.php?image=arrow"
```

Reply:
> 'shell-webadmin.php' was the name of PHP file.

maybe we should shift this rule in PL2